app/components/embed/legacy_pdf_component.html.erb

```diff
@@ -40,8 +40,9 @@
           <% # NOTE: Remove legacy_pdf_viewer.js file when this component is removed %>
           <%= javascript_pack_tag('legacy_pdf_viewer') %>
       <% else %>
-          <div style="padding: 10%;">
-            <strong><%= I18n.t('restrictions.not_accessible') %></strong>
+          <div class="authLinkWrapper">
```

Comment:
Can we extract this whole chunk to a new component and share it between the LegacyPdf, Media and Pdf components?

Reply:
Yes, good idea.  Done.